packages/basic/src/typography.scss

```
@@ -0,0 +1,383 @@
+    border-radius: $pre-border-radius;
+    /* CSS variables --prosemirror-highlight and --prosemirror-highlight-bg are set by package `prosemirror-highlight` */
+    background-color: var(--prosemirror-highlight-bg, inherit);
+    color: var(--prosemirror-highlight, inherit);
+  }
+
+  & pre,
+  & code {
+    white-space: pre;
+    word-break: normal;
+    word-spacing: normal;
+    word-wrap: normal;
```

Stylelint (17.4.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)